package.json

```diff
@@ -13,6 +13,7 @@
     "i18next": "~19.4.0",
     "i18next-browser-languagedetector": "~4.1.0",
     "i18next-xhr-backend": "~3.2.2",
+    "mocha": "~7.1.1",
```

Comment:
Why adding mocha? We already use Create React App integrated test runner (Jest).

Reply:
Aha, I see.   I got confused a bit mixed up reading up on testing nodejs.  Thanks for that...I'll take out the mocha dependency in my next commit.